src/BuildExecutable.jl

```diff
@@ -107,8 +111,9 @@ function build_executable(exename, script_file, targetdir=nothing, cpu_target="n
     emit_cmain(cfile, exename, targetdir != nothing)
     emit_userimgjl(userimgjl, script_file)
 
-    println("running: $(julia) $(build_sysimg) $(sys.buildfile) $(cpu_target) $(userimgjl) --force")
-    run(`$(julia) $(build_sysimg) $(sys.buildfile) $(cpu_target) $(userimgjl) --force`)
+    empty_cmd_str = ``
```

Comment:
Sorry, I tried and failed, the empty should be a named variable, 
```
 run(`some cmd $(debug ? "--debug" : `` )`)
```
does not work, because julia thinks 
```
`some cmd $(debug ? "--debug" : `
```
is a cmd string, this will cause a syntax error.

Maybe Parser of Julia should deal the string/cmd interpolation, but not parse it in the runtime.